final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,232 @@
+"""Main module of the program"""
+
+import sys
+import os
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(this_directory)
+
+
+import args_parser
+import rss_parser
+import other
+import json_converter
+import converter
+import logs
+import news
+import ast
+
+
+def main():
+
+    """This is a main function"""
+    print('============================== RSS Reader ==================================')
+    logs.new_session()
+    commands = args_parser.get_parse()
+
+    while True:
```

Comment:
It is better to just remove user input above and get rid of this `while` completely. This is not how app is supposed to work

Reply:
The structure of the program was fixed, there is no more while cycle. It will be updated during the next commit.